homeassistant/components/ozw/entity.py

```diff
@@ -25,6 +25,7 @@
 from .discovery import check_node_schema, check_value_schema
 
 _LOGGER = logging.getLogger(__name__)
+OZW_READY_STATES = {st.value for st in OZW_READY_STATES}
```

Comment:
Maybe not overwrite the same constant?
```suggestion
OZW_READY_STATES_VALUES = {st.value for st in OZW_READY_STATES}
```